#ifdef HAL_ESP32_HAL_H_

with the latest update of PlatformIO I was no longer able to compile the code. The problem was the statement 
`#ifdef HAL_ESP32_HAL_H_`
in the file OneBitDisplay.cpp.

This is te error-message:

> Compiling .pio/build/usb/lib0c7/OneBitDisplay/OneBitDisplay.cpp.o
.pio/libdeps/usb/OneBitDisplay/src/OneBitDisplay.cpp:51:23: fatal error: BLEDevice.h: No such file or directory

I had to undefine it explicitly. Then it compiles and the code worked.

With v1.5.0 it does not work either for me.

`.pio/libdeps/usb/OneBitDisplay@1.5.0/src/OneBitDisplay.cpp:47:23: fatal error: BLEDevice.h: No such file or directory`

--- a/platformio.ini
+++ b/platformio.ini
@@ -109,6 +109,7 @@ build_flags_all =
     -mfix-esp32-psram-cache-issue
 
 [env]
+lib_ldf_mode = deep ; #632 Fixes compiler error with OneBitDisplay library
 framework = arduino
 board = esp32dev
 board_build.partitions = min_spiffs.csv

